model/src/main/kotlin/config/ScanStorageConfiguration.kt

```diff
@@ -78,9 +78,9 @@ data class PostgresStorageConfiguration(
     val connection: PostgresConnection,
 
     /**
-     * The way that scan results are stored, defaults to [StorageType.PACKAGE_BASED].
+     * The way that scan results are stored, defaults to [StorageType.PROVENANCE_BASED].
      */
-    val type: StorageType = StorageType.PACKAGE_BASED
+    val type: StorageType = StorageType.PROVENANCE_BASED
```

Comment:
Seems I was a bit faster ;-) https://github.com/oss-review-toolkit/ort/pull/6074

Reply:
I missed the outdated docs during the review, though, sorry for that. So there's still value in @mnonnenmacher's change.